apps/graphql/src/apps/location/dataloaders/LocationsByID.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import stringify from 'json-stable-stringify';
+import { OptimisticDataloader } from '@kiwicom/graphql-utils';
+
+import fetch from '../../../services/fetch/tequilaFetch';
+import { type Location } from '../Location';
+import sanitizeLocations from '../helpers/sanitizeLocations';
+
+export type LocationInputID = {|
+  +code: string,
+|};
+
+const fetchLocations = async (params: $ReadOnlyArray<LocationInputID>) => {
+  const results = await fetch(
+    `/locations/id?limit=${params.length}&` +
+      params
+        .map((param: LocationInputID) => {
+          if (param.code !== undefined) {
```

Comment:
Not sure we need this param.code !== undefined anymore

Reply:
We don't, on `RouteStop` the `code` can't be undefined